Source/Orts.Formats.Msts/CabViewFile.cs

```diff
@@ -1293,8 +1312,8 @@ protected void ParseCustomParameters(STFReader stf)
     #region other controls
     public class CVCSignal : CVCDiscrete
     {
-        public CVCSignal(STFReader inf, string basepath)
-            : base(inf, basepath)
+        public CVCSignal(STFReader inf, string basepath, DiscreteStates discreteStates)
+            : base(inf, basepath, discreteStates)
```

Comment:
The name of the parameter should be singular.

Reply:
Yes, of course it should.

Good catch.